src/nlp.jl

```diff
@@ -619,7 +617,39 @@ function MOI.eval_constraint_jacobian(d::NLPEvaluator, J, x)
     return
 end
 
+function MOI.eval_constraint_jacobian_product(d::NLPEvaluator,
+                                              Jv::AbstractVector{Float64},
+                                              x::AbstractVector{Float64},
+                                              v::AbstractVector{Float64})
+    fill!(Jv, 0.0)
+    jac_struct = MOI.jacobian_structure(d)
+    nnzj = length(jac_struct)
```

Comment:
Use underscores to separate words in names: http://www.juliaopt.org/JuMP.jl/dev/style/#Use-of-underscores-within-names-1. I'd suggest naming this `nnz_jac`.